### PR TITLE
Extends only

### PR DESCRIPTION
**Pull request by drakeltheryuujin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,237 +1,61 @@
...
-`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
+that you can create classes and build methods that can perform
+actions on instance data, or specific to the class. What if you have
+classes that exhibit many of the same behaviors, such as `Cat`, `Dog`,
 and `Bird`, which all have a method for `speak`? In JavaScript, you can
-create "child" object `class`es that inherit methods from their "parent"
-`class`es. In this lesson, we'll discuss 2 ways of _extending_ functionality
-to other `class`es.
+create "child" object classes that inherit methods from their "parent"
+classes. In this lesson, we'll discuss 2 ways of _extending_ functionality
```
Comment (maxwellbenton) [inline README.md]: 1 way

File: README.md [hunk 2]
```diff
@@ -1,237 +1,61 @@
...
 
 ## Introduction
 
 In JavaScript, as in other Object Oriented languages, we've learned
-that you can create `class`es and build methods that can perform
-actions on instance data, or specific to the `class`. What if you have
-`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
+that you can create classes and build methods that can perform
+actions on instance data, or specific to the class. What if you have
+classes that exhibit many of the same behaviors, such as `Cat`, `Dog`,
```
Comment (maxwellbenton) [inline README.md]: I think it may be good to have a code snippet for Cat Dog and Bird classes here to refresh student's memory

File: README.md [hunk 3]
```diff
@@ -1,237 +1,61 @@
...
-  constructor(name) {
-    this.name = name;
-  }
-    crawl(){
-      return `${this.name} crawls away`
-    }
-}
+let dog = new Dog("Shadow");
+let cat = new Dog("Missy");
+let bird = new Dog("Tiki");
```
Comment (maxwellbenton) [inline README.md]: Cat and Bird classes

File: README.md [hunk 4]
```diff
@@ -1,237 +1,61 @@
...
 
-class Lizard extends LeggedReptile {}
-class Snake extends Reptile {}
+dog.speak(); // Shadow makes a loud sound!
+cat.speak(); // Missy makes a loud sound!
+bird.speak(); // Tiki makes a loud sound!
 ```
```
Comment (maxwellbenton) [inline README.md]: This is good. I think though that this lab can include an example of a child with a unique method as well as the inherited `speak`. Just to make it clearer that extension means you can inherit a `clas` and then build additions
in a child

Comment (maxwellbenton) [reply README.md]: By an extra method, I mean something entirely unrelated to the parent (so no need for super).

Like `Bird` can have a `get fly() {}`. Bird can still use `speak` but now has its own functionality

Comment (drakeltheryuujin) [reply README.md]: gotcha

File: README.md [hunk 5]
```diff
@@ -1,237 +1,61 @@
...
 ```js
 class Pet {
   constructor(name) {
     this.name = name;
   }
-}
 
-// Inherits from Pet
-class Dog extends Pet {
   speak() {
```
Comment (maxwellbenton) [inline README.md]: I've updated some the content on method types and getters/setters. The suggestion I've added is to use getters and setters whenever a class is dealing with input or output (get and set define the "public menu" of a class). 

If we can, let's stick to using getters and setters in examples. Class methods are sort of downgraded to only function internally as helper methods in this design.

**Pull request closed without merging**